src/Compilers/Core/CodeAnalysisTest/Diagnostics/OperationTests.cs

```diff
@@ -104,5 +106,42 @@ public void IDynamicObjectCreationExpression_PublicExtensionMethodTests()
 
             TestCore(createDynamicExpression);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestFlowAnalysisFeatureFlag()
+        {
+            var source = @"
+class C
+{
+    void M()
+    {
+    }
+}";
+            var tree = CSharpSyntaxTree.ParseText(source);
+            TestFlowAnalysisFeatureFlagCore(tree, expectException: true);
+
+            var options = tree.Options.WithFeatures(new[] { new KeyValuePair<string, string>("flow-analysis", "true") });
```

Comment:
I'm assuming feature flags are case- and whitespace-sensitive. Is that worth testing?

Reply:
Actually, they are case-insensitive. I have added verification in the unit test.